### PR TITLE
disable connection uses  rejection modal

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**
image

### CODE REVIEW

**Comment by gabrielsaruhashi** [comment]
@cshdllr 
took the liberty to also update the connection drawer with this flow

**Comment by cshdllr** [comment]
Content nits...
1. Can we please put "Not an approved visitor" at the top of the list since it is overwhelmingly the #1 rejection reason?
2. Can we use sentence case for the options? (like below)

• Not an approved visitor
• Photo of ID is blurry or cut off
• Incomplete information
• Other

File: src/components/Drawer/ConnectionDetailDrawer.tsx [hunk 1]
```diff
@@ -62,22 +78,34 @@ const ApproveRejectButton: React.FC<ApproveRejectButtonProps> = ({
...
+                      statusDetails: feedback,
+                    },
+                  },
+                })
+              }
+              onClose={() => setIsRejecting(false)}
+              connection={connection}
+              facility={facility}
+            />
+          ) : (
```
Comment (cainlevy) [inline src/components/Drawer/ConnectionDetailDrawer.tsx:99]: i think typically we would continue rendering the button

**Review by cainlevy — APPROVED** [review]

File: src/components/Drawer/ConnectionDetailDrawer.tsx (hunk 1 above)
Comment (gabrielsaruhashi) [reply src/components/Drawer/ConnectionDetailDrawer.tsx]: oop, was an oversight

**Pull request merged**